Jenkinsfile_CNP

```diff
@@ -34,4 +34,43 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 withPipeline(type, product, component) {
     loadVaultSecrets(secrets)
     enableDbMigration()
-}
+    
```

Comment:
I think you want to add code here https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/GradleBuilder.groovy#L23

That's a line I added to make sure we get checkstyle reports